bitnami/grafana-operator/templates/grafana.yaml

```diff
@@ -120,6 +120,9 @@ spec:
     {{- end }}
     {{- if or .Values.commonAnnotations .Values.grafana.ingress.annotations }}
     annotations:
+      {{- if .Values.grafana.ingress.certManager }}
```

Comment:
Hi,
  Take into account that this "if" is inside of the "if" in line 121 and this should be independent.